### PR TITLE
ZDI assigns the following CVEs:

### PR DESCRIPTION
**Pull request by zdi-team**
M  2020/15xxx/CVE-2020-15420.json
M  2020/15xxx/CVE-2020-15421.json
M  2020/15xxx/CVE-2020-15422.json
M  2020/15xxx/CVE-2020-15423.json
M  2020/15xxx/CVE-2020-15424.json
M  2020/15xxx/CVE-2020-15425.json
M  2020/15xxx/CVE-2020-15426.json
M  2020/15xxx/CVE-2020-15427.json
M  2020/15xxx/CVE-2020-15428.json
M  2020/15xxx/CVE-2020-15429.json
M  2020/15xxx/CVE-2020-15430.json
M  2020/15xxx/CVE-2020-15431.json
M  2020/15xxx/CVE-2020-15432.json
M  2020/15xxx/CVE-2020-15433.json
M  2020/15xxx/CVE-2020-15434.json
M  2020/15xxx/CVE-2020-15435.json
M  2020/15xxx/CVE-2020-15606.json
M  2020/15xxx/CVE-2020-15607.json
M  2020/15xxx/CVE-2020-15608.json
M  2020/15xxx/CVE-2020-15609.json
M  2020/15xxx/CVE-2020-15610.json
M  2020/15xxx/CVE-2020-15611.json
M  2020/15xxx/CVE-2020-15612.json
M  2020/15xxx/CVE-2020-15613.json
M  2020/15xxx/CVE-2020-15614.json
M  2020/15xxx/CVE-2020-15615.json
M  2020/15xxx/CVE-2020-15616.json
M  2020/15xxx/CVE-2020-15617.json
M  2020/15xxx/CVE-2020-15618.json
M  2020/15xxx/CVE-2020-15619.json
M  2020/15xxx/CVE-2020-15620.json
M  2020/15xxx/CVE-2020-15621.json
M  2020/15xxx/CVE-2020-15622.json
M  2020/15xxx/CVE-2020-15623.json
M  2020/15xxx/CVE-2020-15624.json
M  2020/15xxx/CVE-2020-15625.json
M  2020/15xxx/CVE-2020-15626.json
M  2020/15xxx/CVE-2020-15627.json
M  2020/15xxx/CVE-2020-15628.json

### CODE REVIEW

File: 2020/15xxx/CVE-2020-15420.json [hunk 1]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15420.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15421.json [hunk 2]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15421.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15422.json [hunk 3]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15422.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15423.json [hunk 4]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15423.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15424.json [hunk 5]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15424.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15425.json [hunk 6]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15425.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15426.json [hunk 7]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15426.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15427.json [hunk 8]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15427.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15428.json [hunk 9]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15428.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15429.json [hunk 10]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15429.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15430.json [hunk 11]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15430.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15431.json [hunk 12]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15431.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15432.json [hunk 13]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15432.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15433.json [hunk 14]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15433.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15434.json [hunk 15]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15434.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15435.json [hunk 16]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2020/15xxx/CVE-2020-15435.json:1]: CNA(s) of user (zdi) do not match owning CNA.

File: 2020/15xxx/CVE-2020-15420.json (hunk 1 above)
Comment (zdi-team) [reply 2020/15xxx/CVE-2020-15420.json:1]: @cve-team all of these are assigned to ZDI, see #4393

**Comment by cve-team** [comment]
Hi ZDI team, sorry for the mixup. A coordinator mistook these for Trend Micro based on the email address of the submitter who registered. I obtained the list of ones assigned and have transferred them over to ZDI. The pull request might need to be closed and resubmitted though. Apologies for any confusion/frustration and thanks for your patience on this matter.

**Comment by zdi-team** [comment]
@cve-team thanks for the merge, I hadn't yet had a moment to poke the PR w/another commit

**Pull request merged**